src/OrchardCore.Themes/TheAdmin/Views/Layout.cshtml

```diff
@@ -14,8 +14,9 @@
     <!-- Bootstrap CSS -->
     <link rel="stylesheet" href="/TheAdmin/Styles/TheAdmin.css">
 
-    <script asp-name="font-awesome" use-cdn="true" at="Foot" version="5"></script>
-    <script asp-name="font-awesome-v4-shims" use-cdn="true" at="Foot" version="5"></script>
+    <script asp-name="font-awesome-v4-shims" use-cdn="false"></script>
```

Comment:
Added twice?

Reply:
sorry , I will delete it , it's a cdn test for font-awesome